src/components/dropdown-filter-ajax/dropdown-filter-ajax.stories.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
```

Comment:
Large list of propTypes here without description.

![image](https://user-images.githubusercontent.com/26925206/53957434-42d5d600-40d6-11e9-830d-06d7b1bda92b.png)

Reply:
This is one that we've seen in a few of the Storybook PRs; the component is wrapped in several HOCs before it's exported, which is preventing Storybook from being able to get the info it needs. I'm suggesting we ignore these instances in favour of fixing when the components are refactored.